acceptable/tests/test_util.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import unicode_literals
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+# Copyright 2017 Canonical Ltd.  This software is licensed under the
```

Comment:
2018

Reply:
Right, I'll update them all in a separate PR